torch/quantization/fx/_equalize.py

```diff
@@ -261,11 +266,40 @@ def get_weight_eq_obs(
         return op_node, weight_eq_obs
 
     elif op_node.op == 'call_function':
-        # TODO
-        return None, None
+        assert(isinstance(op_node.args[1], Node))
+        weight_observer_nodes = collect_producer_nodes(op_node.args[1])
+        if weight_observer_nodes is None:
+            return None, None
+
+        for weight_node in weight_observer_nodes:
+            if weight_node.op == 'call_module' and \
+               isinstance(modules[str(weight_node.target)], _WeightEqualizationObserver):
+
+                weight_eq_obs = modules[str(weight_node.target)]
+                assert(isinstance(weight_eq_obs, _WeightEqualizationObserver))
+                return op_node, weight_eq_obs
 
+        return None, None
     return None, None
 
+def maybe_get_weight_eq_obs_node(op_node: Node, modules: Dict[str, nn.Module]) -> Optional[Node]:
+    """ Given the operation node, we want to find its previous weight
```

Comment:
Should the comment be "find its weight equalization observer" as opposed to "its previous ..."?